src/main/java/com/shaft/tools/io/internal/AllureManager.java

```diff
@@ -153,11 +160,12 @@ private static void createAllureListenersMetaFiles() {
                 "io.qameta.allure.listener.StepLifecycleListener", "io.qameta.allure.listener.TestLifecycleListener").forEach(fileName -> FileActions.getInstance(true).writeToFile(Properties.paths.services(), fileName, "com.shaft.listeners.AllureListener"));
     }
 
-    private static void writeAllureReport() {
+    public static void writeAllureReport() {
         String commandToCreateAllureReport;
         allureBinaryPath = allureExtractionLocation + "allure-" + SHAFT.Properties.internal.allureVersion()
                 + "/bin/allure";
         String outputDirectory = System.getProperty("user.dir") + File.separator + "target" + File.separator + allureReportPath;
+        allureOutPutDirectory = outputDirectory;
```

Comment:
We don't need the local variable, and we don't need to assign value inside this method for the global variable, we can assign on the same declaration line since the path is static.